api4/user_test.go

```diff
@@ -2281,34 +2280,43 @@ func TestUpdateUserPassword(t *testing.T) {
 
 	password := "newpassword1"
 	pass, resp := th.Client.UpdateUserPassword(th.BasicUser.Id, th.BasicUser.Password, password)
+	th.App.InvalidateCacheForUser(th.BasicUser.Id)
```

Comment:
Shouldn't this invalidation happen in the server side app layer ? I see `InvalidateCacheForUser` being called in several places in app/user.go. Perhaps it is missing this call in the `UpdatePassword` method ?

I mean what happens if an actual client calls this method ? They should not need to manually invalidate the cache right ?

Reply:
Hum, you're right. I've misread that part thinking it was just an internal method. I'm going to check. Good point. Thank you